include/mbedtls/oid.h

```diff
@@ -466,6 +471,25 @@ typedef struct mbedtls_oid_descriptor_t {
  */
 int mbedtls_oid_get_numeric_string(char *buf, size_t size, const mbedtls_asn1_buf *oid);
 
+/**
+ * \brief           Translate a string containing a dotted-decimal
+ *                  representation of an ASN.1 OID into its encoded form
+ *                  (e.g. "1.2.840.113549" into "\x2A\x86\x48\x86\xF7\x0D").
+ *                  On success, this function allocates oid->buf from the
+ *                  heap. It must be freed by the caller using mbedtls_free().
+ *
+ * \param oid       mbedtls_asn1_buf to populate with the DER-encoded OID
```

Comment:
Should `mbedtls_asn1_buf` be `#mbedtls_asn1_buf`? (sorry, I know I only looked at this yesterday) - if this is the only thing, don't bother with it